test/e2e/fixtures/README.md

```diff
@@ -1,10 +1,3 @@
-# End-to-end tests
-
-This directory contains the fixture data used to bootstrap the individual e2e tests. Each sub-directory contains
-one thing:
-
-1. A `state.json` file that represents a the saved state for the extension (see _Generating fixture data_ below)
-
 ## Generating fixture data
```

Comment:
Maybe this should be made the main title of this file, and moved into `docs/` as an independent article on generating fixture data. It's not very discoverable left here in this directory